### ISSUE TITLE
Implement FC layer visualizer

### ISSUE THREAD

**Issue by NMZ0429** [NMZ0429/visualizer#1]
# Objective

Visualize how output logits vary depending on inputs.

# TODO
- [ ] Implement the function to store result logits from test step
- [ ] Implement visualizing function in streamlit
- [ ] Pass the result from model to app
- [ ] Visualize it
- [ ] Try single inference and embed it to the plot.
